setup.py

```diff
@@ -55,7 +55,7 @@
     install_requires=[
         "argparse>=1.2.1,<1.3.0",
         "pyres>=1.5,<1.6",
-        "Pillow>=4.3.0,<5.2.0",
```

Comment:
why not 4.3.0+?

Reply:
I disagree with the strategy of supporting older versions of Pillow. It seems like an unnecessary maintenance burden. There are strategies like `virtualenv` which should ease pain of running on an application server with multiple apps that require pillow. Not to mention Docker.

Is there there some particular reason _to support_ older version of Pillow?

Reply:
Pillow-simd is not there yet.
BC break :man_shrugging:?

Reply:
@kkopachev pillow-simd appears to have a 6.0.0 release https://github.com/uploadcare/pillow-simd/releases/tag/6.0.0 ... unless I am missing something.

Reply:
That's original pillow tag. I guess it's because repo is forked, so it replicates original tags as well.
Latest release was 5.3.0-post1 https://pypi.org/project/Pillow-SIMD/#history

Reply:
@kkopachev Oh, hmm, that's funny. Ok. Well I guess that depends on how big of a deal it is to support pillow-simd explicitly.

In addition to the maintenance burden it may take a bit of doing to port something like this https://github.com/uploadcare/pillow-simd/commit/83cdbc0de5f77d5c29cb15370e8733e8b24bd08b (that you pointed out earlier) to thumbor to use as a fallback. I understand approximately 0% of that code so I'm not real super confident doing it myself. If you are keen to port that over as part of this work I certainly wouldn't object. You interested?

Reply:
@kkopachev I opened a [ticket with pillow-simd](https://github.com/uploadcare/pillow-simd/issues/54) for a 6.0.0 release request + context.

I think we should proceed with upgrading Thumbor's pillow version to 6.0.0 only. The pillow-simd project is great and I'd honestly love to have support for our own cluster. But the project is naturally going to lag a bit in release cycles. I think it's ok if a new version of Thumbor ships without a 6.0.0 pillow-simd release. It's imperfect but those who have servers blowing up probably care more about stability. Those who don't have EXIF/blowing up issues can stick to their current thumbor release as the new release is feature equivalent.

Another option is to add support for pillow < 6.0.0 _after_ we get this particular issue fixed. That could be it's own effort.

Does that seem reasonable to you?

Reply:
That seems reasonable.

Reply:
@kkopachev I thought you might like to know pillow-simd has a 6.0.0 release now https://github.com/uploadcare/pillow-simd/issues/54 Pretty rad.